lib/util/git/index.ts

```diff
@@ -779,10 +779,38 @@ export async function deleteBranch(branchName: string): Promise<void> {
   delete config.branchCommits[branchName];
 }
 
-export async function mergeBranch(
-  branchName: string,
-  localOnly = false
-): Promise<void> {
+export async function mergeToLocal(refSpecToMerge: string): Promise<void> {
+  let status: StatusResult | undefined;
+  try {
+    await syncGit();
+    await writeGitAuthor();
+    await git.reset(ResetMode.HARD);
+    await gitRetry(() =>
+      git.checkout([
+        '-B',
+        config.currentBranch,
+        'origin/' + config.currentBranch,
+      ])
+    );
+    status = await git.status();
+    await fetchRevSpec(refSpecToMerge);
+    await gitRetry(() => git.merge(['FETCH_HEAD']));
```

Comment:
Why did you choose this instead of simple git merge <branchName>?
Is it to be safe and always get the latest commit?

Thanks for the PR. I wasn't aware of the effect on Gerrit due to my changes.

Reply:
> Why did you choose this instead of simple git merge ? Is it to be safe and always get the latest commit?

What do you mean?
I'm fetching the "branch" (or more general a refspec) and then merge these fetched ref.

Reply:
My bad. I meant why not `git merge <branchName>` like before?

Reply:
> My bad, some part gotcleared while editing. I meant why not `git merge <branchName>` like before?

because i want to allow the fetch and merge of individual refspecs and not only "branches" (refs/heads).